content/specs/lexicon.md

```diff
@@ -315,6 +316,14 @@ Invalid examples:
 Flexible to any URI schema, following the generic RFC-3986 on URIs. This includes, but isn’t limited to: `did`, `https`, `wss`, `ipfs` (for CIDs), `dns`, and of course `at`.
 Maximum length in Lexicons is 8 KBytes.
 
+### `language`
+
+An [IETF Language Tag](https://en.wikipedia.org/wiki/IETF_language_tag) string, compliant with [BCP 47](https://www.rfc-editor.org/info/bcp47), defined in [RFC 5646](https://www.rfc-editor.org/rfc/rfc5646.txt) ("Tags for Identifying Languages"). This is the same standard used to identify languages in HTTP, HTML, and other web standards.
```

Comment:
One little note here— the spec has two classes of conformance ([link](https://www.rfc-editor.org/rfc/rfc5646.html#section-2.2.9)): "well-formed" and "valid".  One of the requirements for valid tags is that they align with the living IANA language subtag registry.  For that reason, our current validation only looks for well-formedness.  I don't think that's set in stone, but that's the current state of things 👍 

```
   A tag is considered "well-formed" if it conforms to the ABNF
   Section 2.1.  Language tags may be well-formed in terms of syntax
   but not valid in terms of content.  However, many operations
   involving language tags work well without knowing anything about the
   meaning or validity of the subtags.
```

Reply:
I think we should stick with "well-formed" as the string format requirement, a say that folks should ignore languages which are "well-formed" but not "valid". My read is that "valid" will expand over time (with additional language codes, suffixes, etc), but "well-formed" is more stable.

Reply:
That sounds good 👍